### PR TITLE
Improved info on giving access to cloud features

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

**Comment by JordyGit** [comment]
All DataMiner rights/permissions that the linked DataMiner user has are still applied in the web apps. I think there are permissions to access web apps? so these would also still apply.

**Review by baptistepa — APPROVED** [review]

File: user-guide/Advanced_Modules/Job_Manager/Accessing_the_jobs_app.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+http://[DMA]/jobs
+https://[DMA]/jobs
+```
+
+> [!CAUTION]
+> If you use a DataMiner version prior to DataMiner 10.1.7, we strongly advise that you use HTTPS when you use DataMiner client applications over public internet. If you do not do so, all information – including logon credentials – is sent as plain, unencrypted text over the internet. From DataMiner 10.1.7 onwards, client-server communication is encrypted by default. See also: [Setting up HTTPS on a DMA](xref:Setting_up_HTTPS_on_a_DMA).
+
+## Accessing the Jobs app via the DataMiner Cloud Platform
+
+If you have been granted [remote access](xref:Controlling_remote_access) to a cloud-connected DataMiner System as a guest, you can connect as follows:
```
Comment (JordyGit) [inline user-guide/Advanced_Modules/Job_Manager/Accessing_the_jobs_app.md:19]: i would remove the 'as a guest' in all the web apps pages, the users need a DCP and DataMiner account so they are not really a guest anymore (?)

```suggestion
If you have been granted [remote access](xref:Controlling_remote_access) to a cloud-connected DataMiner System, you can connect as follows:
```

File: user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md [hunk 2]
```diff
@@ -4,11 +4,16 @@ uid: Controlling_remote_access
 
 # Controlling remote access with the Admin app
 
-When a DMS is cloud-connected, you can use the URL created for you during registration (e.g. ``https://ziine-skyline.on.dataminer.services/``) to access the DataMiner web apps from anywhere.
+When a DMS is cloud-connected, you can use the URL created for you during registration (e.g. ``https://ziine-skyline.on.dataminer.services/``) to access the DataMiner web apps from anywhere. See for example [Accessing the Monitoring app via the DataMiner Cloud Platform](xref:Accessing_the_Monitoring_app#accessing-the-monitoring-app-via-the-dataminer-cloud-platform).
 
 ## Finding the remote access URL
 
-After you have registered, you can find the URL on the dataminer.services home page. In the DataMiner Systems section, the URL is displayed right underneath the name of the connected DataMiner System. You can also directly navigate to the URL from this section.
+After you have registered, you can find the URL on the *Overview* page for the DataMiner System in the Cloud Admin app. The URL is displayed right underneath the name of the connected DataMiner System. You can also directly navigate to the URL from this section.
```
Comment (JordyGit) [inline user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md:11]: To be consistent , I think we have used DCP Admin before (and in the UI on https://dataminer.services) 
```suggestion
After you have registered, you can find the URL on the *Overview* page for the DataMiner System in the DCP Admin app. The URL is displayed right underneath the name of the connected DataMiner System. You can also directly navigate to the URL from this section.
```

File: user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md [hunk 3]
```diff
@@ -28,3 +32,6 @@ If you have the Owner or Admin role on DCP, you can enable or disable the remote
 1. Click the *Edit* button at the top of the page.
 
 1. Set the *Remote Access* setting to *On* or *Off*, depending on whether you want this to be enabled or not.
+
+> [!NOTE]
+> Users will only be able to use the remote access feature if they have been given access to cloud features. See [Controlling user access to cloud features](xref:Giving_users_access_to_cloud_features). They will also need to have the [General > DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission, as well as any other user permissions required to access specific apps, e.g. [Modules > User-definable apps > View apps](xref:DataMiner_user_permissions#modules--user-definable-apps--view-apps) for the Low-Code Apps.
```
Comment (JordyGit) [inline user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md:37]: Isn't this better?
```suggestion
> Users will only be able to use the remote access feature if they have been given access. See [Controlling user access to cloud features](xref:Giving_users_access_to_cloud_features). They will also need to have the [General > DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission, as well as any other user permissions required to access specific apps, e.g. [Modules > User-definable apps > View apps](xref:DataMiner_user_permissions#modules--user-definable-apps--view-apps) for the Low-Code Apps.
```

**Review by JordyGit — CHANGES_REQUESTED** [review]

File: user-guide/Cloud_Platform/CloudAdminApp/DataMiner_Admin_app.md [hunk 4]
```diff
@@ -10,12 +10,10 @@ The Admin app is intended for the management of your cloud-connected DataMiner S
 
 - [Getting your organization verified](xref:CloudConnectionVerification)
 
-- [Giving users access to cloud features](xref:Giving_users_access_to_cloud_features)
+- [Controlling remote access](xref:Controlling_remote_access)
```
[the remark this answers is not in the document]
Comment (MariekeGO) [reply user-guide/Cloud_Platform/CloudAdminApp/DataMiner_Admin_app.md:15]: Indeed, that would be another good addition. However, I'd postpone that for a later update, so we can get this one out there as soon as possible.

**Review by JordyGit — APPROVED** [review]

**Pull request merged**